server/py/framework/service/__init__.py

```diff
@@ -119,7 +119,7 @@ def is_forwarded_request(self, request: fastapi.Request) -> bool:
 
     @abstractmethod
     async def _move_service_to_online(self):
-        pass
+        await run_async_function_with_new_db_session(self._sync_system_metadata)
```

Comment:
You're defeating the purpose of Abstract methods - they should either be completely empty (just `pass`) or raise `NotImplementedError`.
either remove the wrapper or call this in each implementation

Reply:
oh I completely missed that decorator